2-mandatory/6-journey-planner.js

```diff
@@ -123,6 +129,13 @@ function getLocationName() {}
 */
 function journeyPlanner(locations, transportMode) {
   // Implement the function body
+  let result = [];
+  for (let i = 0; i < locations.length; i++) {
+    if (locations[i].includes(transportMode)){
+      result.push(locations[i][0])
+    }
+  }
+  return result
```

Comment:
I think the idea here was to use the functions you wrote earlier (`getLocationName` and `isAccessibleByTransportMode`)

also, you can use` .filter() `and `.map()`